binr/r2r/load.c

```diff
@@ -25,6 +25,12 @@ static char *readline(char *buf, size_t *linesz) {
 	if (end) {
 		*end = '\0';
 		*linesz = end - buf;
+#if __WINDOWS__
+		if (end != buf && *(end - 1) == '\r') {
```

Comment:
we dont like *() in r2land, better use end[-1], smpler and cleaner. also, negative indexing is bad.